src/coreComponents/mesh/NodeManager.cpp

```diff
@@ -105,6 +105,44 @@ void NodeManager::buildSets( CellBlockManagerABC const & cellBlockManager,
   } );
 }
 
+void NodeManager::setIsExternal( FaceManager const & faceManager )
```

Comment:
what is the difference between this method and `NodeManager::setDomainBoundaryObjects`?  I may not understand what external mean here coz it sounds to me the same as being on the boundary.